subo/command/compute_create_token.go

```diff
@@ -48,7 +48,9 @@ func ComputeCreateTokenCommand() *cobra.Command {
 
 // getVerifierCode gets the 6-character code from stdin.
 func getVerifierCode() (string, error) {
-	fmt.Print("A verification code was sent to your email address. Enter the code to continue: ")
+	fmt.Print("A verification code was sent to your email address. " +
+		"Enter the code to continue, " +
+		"and your token will print below (keep it safe!): ")
```

Comment:
Perhaps we should be specific and say 'your environment token' so that it exactly matches the nomenclature we use in the docs?

Reply:
Also, you can use backticks instead of double-quotes to do multi-line strings in Go, if you wish to do so :)

Reply:
> Perhaps we should be specific and say 'your environment token' so that it exactly matches the nomenclature we use in the docs?

Done!

Reply:
> Also, you can use backticks instead of double-quotes to do multi-line strings in Go, if you wish to do so :)

Thanks! Is there a standard around that for us?

Reply:
Nope! It's a Go language feature. If you want it to be one line, use double quotes, if you want it to be multi-line, use backticks.